docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,121 +4,113 @@
+. Specify your server certificate and private key in `kibana.yml`:
 +
 --
+If your certificate/key are contained in a PKCS #12 keystore, specify it like so:
+
 [source,yaml]
 --------------------------------------------------------------------------------
-server.ssl.enabled: true
-server.ssl.key: /path/to/your/server.key
-server.ssl.certificate: /path/to/your/server.crt
+server.ssl.keystore.path: "/path/to/your/keystore.p12"
+server.ssl.keystore.password: "optional decryption password"
+--------------------------------------------------------------------------------
+
+Otherwise, if your certificate/key are in PEM format, specify them like so:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+server.ssl.certificate: "/path/to/your/server.crt"
+server.ssl.key: "/path/to/your/server.key"
+server.ssl.keyPassphrase: "optional decryption password"
 --------------------------------------------------------------------------------
 
 After making these changes, you must always access {kib} via HTTPS. For example, 
 https://localhost:5601.
 
-// TBD: The reference information for server.ssl.enabled says it "enables SSL for 
-// outgoing requests from the Kibana server to the browser". Do we need to 
-// reiterate here that only one side of the communications is encrypted?
-
 For more information, see <<settings,{kib} configuration settings>>.
 --
 
-. Configure {kib} to connect to {es} via HTTPS:
-+
---
+==== Encrypting traffic between {kib} and {es}
+
 NOTE: To perform this step, you must 
 {ref}/configuring-security.html[enable the {es} {security-features}] or you 
 must have a proxy that provides an HTTPS endpoint for {es}. 
 
---
-
-.. Specify the HTTPS protocol in the `elasticsearch.hosts` setting in the {kib}
-configuration file, `kibana.yml`:
+. Specify the HTTPS URL in the `elasticsearch.hosts` setting in the {kib} configuration file, `kibana.yml`:
 +
 --
 [source,yaml]
 --------------------------------------------------------------------------------
 elasticsearch.hosts: ["https://<your_elasticsearch_host>.com:9200"]
 --------------------------------------------------------------------------------
---
-
-.. If you are using your own CA to sign certificates for {es}, set the
-`elasticsearch.ssl.certificateAuthorities` setting in `kibana.yml` to specify
-the location of the PEM file.
-+
---
-[source,yaml]
---------------------------------------------------------------------------------
-elasticsearch.ssl.certificateAuthorities: /path/to/your/cacert.pem
---------------------------------------------------------------------------------
 
-Setting the `certificateAuthorities` property lets you use the default 
-`verificationMode` option of `full`. 
-//TBD: Is this still true? It isn't mentioned in https://www.elastic.co/guide/en/kibana/master/settings.html
+Using the HTTPS protocol results in a default `elasticsearch.ssl.verificationMode` option of `full`, which utilizes hostname verification.
```

Comment:
@lcawl That statement wasn't entirely accurate, I've updated it here to clarify, and I have also added a matching sentence in the reference for `elasticsearch.hosts`.